src/Wordpress/PluginActions.php

```diff
@@ -52,4 +52,65 @@ public function loginWordPress()
 
         return $response;
     }
+
+    /**
+     * GET /plugin/:zonedId/settings.
+     *
+     * @return mixed
+     */
+    public function getPluginSettings()
+    {
+        // Always return true
+        $value = true;//$this->dataStore->getIpRewrite();
+
+        $response = $this->api->createAPISuccessResponse(
+            array(
+                array(
+                'id' => 'ip_rewrite',
+                'value' => $value,
+                'editable' => true,
+                'modified_on' => '',
+                ),
```

Comment:
Since every individual setting will have the same structure (id, value, editable, modified on) lets create a method that takes in those four parameters as arguments and returns that array as the response.  Default editable to true if its not passed in and default modified on to empty string if its not passed in.  This will make adding new settings easy in the future.